odtp/workflow.py

```diff
@@ -48,12 +48,15 @@ def __init__(self, execution_data, working_path, secrets):
                 step_folder_path = os.path.join(self.working_path, step_name)
                 self.steps_folder_paths.append(step_folder_path)
 
-                image_name = step_name
+                image_name = odtp_utils.get_execution_step_name(
```

Comment:
@caviri Is that correct? so the `image_name` does not have the `step_index` as a parameter any more? 

Currently it is  `1_odtp-pygwalker_v0.1.1` same as the `step_name`. But this would give you `odtp-pygwalker_v0.1.1`: is that correct?

Reply:
Okay this is correct: The image_name will be indeed `odtp-pygwalker_v0.1.1`